images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -139,16 +130,13 @@ $utilitiesList = @(
     (Get-MongodVersion),
     (Get-7zipVersion),
     (Get-BsdtarVersion),
-    (Get-GnuTarVersion)
+    (Get-GnuTarVersion),
+    (Get-HelmVersion),
+    (Get-GPGVersion),
+    (Get-SwitchAudioOsxVersion),
+    (Get-SoxVersion)
 )
 
-if ($os.IsLessThanMonterey) {
-    $utilitiesList += @(
-        (Get-GPGVersion),
-        (Get-HelmVersion)
-    )
-}
-
 if ($os.IsHigherThanMojave -and $os.IsLessThanMonterey) {
     $utilitiesList += @(
         (Get-NewmanVersion)

```

Comment:
Out of curiosity, is newman still unsupported or deprecated?

Reply:
I'm not really sure. I just walked through the list in the workitem and the list of canary tests I need to update =)